config/data_sources/acs_condition.tf

```diff
@@ -0,0 +1,5 @@
+# Create a BigQuery dataset for acs health insurance information
```

Comment:
this comment should indicate this is pulling more than just health insurance from ACS

Reply:
good catch, thanks